### DISCUSSION TITLE
I got trouble when will hide modal not working

### DISCUSSION THREAD

**Discussion by faruuhan** [twbs/bootstrap#36111] [Q&A]
`const modalGetGotcha = (value) => {
    let gotchaSuccessModal = new bootstrap.Modal(document.getElementById("testModal"), {});
    if (value == "open") {
      gotchaSuccessModal.show();
    } else if (value == "close") {
      gotchaSuccessModal.hide();
    }
  };
`

modal show working but when to coditional close modal hide not working, help me

**Comment by gerardnico** [comment]
Have you tried with [getinstance](https://getbootstrap.com/docs/5.0/components/modal/#getinstance) version 5.0
```javascript
const modalGetGotcha = (value) => {
    let modalElement = document.getElementById("testModal");
    let gotchaSuccessModal = Modal.getInstance(modalElement);
    if (value === "open") {
        gotchaSuccessModal.show();
    } else if (value === "close") {
        gotchaSuccessModal.hide();
    }
};
```
